jujugui/static/gui/src/test/test_environment_view.js

```diff
@@ -1537,6 +1537,35 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
       assert.equal('hidy ho', endpointsController);
       view.destroy();
     });
+
+    describe('onboarding integration with the environment', function() {
+      // XXX This test does not run in Phantom, but passes in the browser.
+      // See https://github.com/ariya/phantomjs/issues/12782 for details.
+      // When touching the integrated button code, this should be enabled
+      // and tested, then skipped before proposing or CI will fail.
+      // Makyo - 2015-09-21
+      it.skip('shows/hides the integrated button when a service is added',
```

Comment:
Instead of skipping you can probably grab the browser, and if phantom then return